### ISSUE TITLE
schedule! no method error

### ISSUE THREAD

**Issue by Daniel-ltw** [moove-it/sidekiq-scheduler#152]
NoMethodError: undefined method `schedule!' for Sidekiq:Module
	/usr/local/bundle/gems/sidekiq-scheduler-2.0.20/lib/sidekiq-scheduler/web.rb:11:in `block in registered
	/usr/local/bundle/gems/sidekiq-4.2.5/lib/sidekiq/web/application.rb:266:in `instance_exec'
	/usr/local/bundle/gems/sidekiq-4.2.5/lib/sidekiq/web/application.rb:266:in `block in call'
	/usr/local/bundle/gems/sidekiq-4.2.5/lib/sidekiq/web/application.rb:262:in `catch'
	/usr/local/bundle/gems/sidekiq-4.2.5/lib/sidekiq/web/application.rb:262:in `call'

**Comment by snmgian** [comment]
PR opened: #153 

In the meantime, you can have these requires in your config.ru:

```ruby
require 'sidekiq/web'
require 'sidekiq-scheduler/web'

require 'sidekiq-scheduler'
```

**Comment by snmgian** [comment]
Released version 2.0.21
